features/FEATURE_BLE/ble/GattClient.h

```diff
@@ -635,6 +635,27 @@ class GattClient {
         (void) characteristic;
     }
 
+    /**
+     * Trigger MTU negotiation. This might result in a Gap event onAttMtuChange
+     * being called if MTU changes.
+     *
+     * @note This does not guarantee a change in MTU size. If size remains
+     * unchanged no event will be generated.
+     *
+     * @param connection Connection on which the MTU is to be negotiated.
+     *
+     * @return BLE_ERROR_NONE if the procedure has been launched successfully
+     * otherwise an appropriate error.
+     */
+    virtual ble_error_t negotiateAttMtu(
+        Gap::Handle_t connection
```

Comment:
please use `ble::connection_handle_t`.

Reply:
sorry, used whatever I found being used in the file, should we replace all other Gap::Handle_t?